Src/VimCore/CommandUtil.fs

```diff
@@ -1779,23 +1779,20 @@ type internal CommandUtil
     /// Happens when the middle mouse button is clicked.  Need to paste the contents of the default
     /// register at the current position
     member x.PutAfterCaretMouse() =
-        try
+        match TextViewUtil.GetTextViewLines _textView with

```

Comment:
Consider combining the match clauses.

Reply:
Gotcha, we can match on a tuple because we need both of them to be some value.